allo/backend/hls.py

```diff
@@ -172,12 +182,15 @@ def __init__(
         configs=None,
         func_args=None,
         wrap_io=True,
-    ):
+        custom_bd_tcl="block_design.tcl",
```

Comment:
If you are not using `custom_bd_tcl` in your project or using it in the test, then this argument should be removed.